### DISCUSSION TITLE
[Feature Request][Review view] Search in all files

### DISCUSSION THREAD

**Discussion by davidzambo** [community/community#145473] [Mobile]
### Select Topic Area

General

### Body

When i review a pull request and I find for example a new method, I would like to see where it is used.

When I do the review in the browser, by pressing CMD+F, I can search on the whole page for text occurrences. It would be nice if i could do the same in the GitHub mobile app too.

**Comment by iskae** [comment]
Hi @davidzambo 👋 

Thanks for reaching out! It looks like we already have an internal tracking issue for this, and I will be adding your feedback in there too 🙇 

Please stay tuned for our future releases! 

Cheers
